### PR TITLE
Fixes #777 - Set Fluo version in fluo.properties during build

### LINKED ISSUE #777: Set Fluo version in fluo.properties during build
Currently, the Fluo version is hard coded in fluo.properties and needs to be changed by hand for each release.  This can be easily forgotten.  Instead, the filter tool should be used in the maven assembly plugin to set correct Fluo version when binary tarball is built.

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

**Comment by keith-turner** [comment]
http://stackoverflow.com/questions/4056140/maven-resource-filtering-implications-of-the-symbol-in-resource-files

File: modules/distribution/src/main/config/fluo.properties [hunk 1]
```diff
@@ -56,7 +56,7 @@ fluo.admin.hdfs.root=hdfs://localhost:10000
 # every tablet server.  For the default jars below, `fluo init` will place them
 # in HDFS.  If you add more jars to the classpath below, you will need to
 # add them to HDFS.
-fluo.admin.accumulo.classpath=${fluo.admin.hdfs.root}/fluo/lib/fluo-api-1.0.0-incubating-SNAPSHOT.jar,${fluo.admin.hdfs.root}/fluo/lib/fluo-accumulo-1.0.0-incubating-SNAPSHOT.jar
+fluo.admin.accumulo.classpath=${fluo.admin.hdfs.root}/fluo/lib/fluo-api-${project.version}.jar,${fluo.admin.hdfs.root}/fluo/lib/fluo-accumulo-${project.version}.jar
```
Comment (mikewalch) [inline modules/distribution/src/main/config/fluo.properties]: Fixed in c95f4ba6cb30c

File: modules/distribution/pom.xml [hunk 2]
```diff
@@ -63,6 +63,9 @@
     <plugins>
       <plugin>
         <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <escapeString>\</escapeString>
```
Comment (ctubbsii) [inline modules/distribution/pom.xml]: This config should be moved into the execution-specific config a few lines lower.

File: modules/distribution/src/main/assembly/bin.xml [hunk 3]
```diff
@@ -94,4 +94,11 @@
       </includes>
     </fileSet>
   </fileSets>
+  <files>
+    <file>
+      <source>src/main/config/fluo.properties</source>
+      <outputDirectory>conf/examples</outputDirectory>
```
Comment (mikewalch) [inline modules/distribution/src/main/assembly/bin.xml]: Fixed in 84e90dda52d

File: modules/distribution/pom.xml (hunk 2 above)
Comment (mikewalch) [inline modules/distribution/pom.xml]: Fixed in edd4e1f118e46

**Pull request merged**